package.json

```diff
@@ -156,7 +156,7 @@
 		"clean:packages": "rimraf ./packages/*/build ./packages/*/build-module ./packages/*/build-style ./packages/*/node_modules",
 		"prebuild:packages": "npm run clean:packages && lerna run build",
 		"build:packages": "node ./bin/packages/build.js",
-		"build": "npm run build:packages && wp-scripts build",
+		"build": "npm run build:packages && wp-scripts build --no-babel",
```

Comment:
I personally don't like the `no-babel` option. I wonder if we're not doing ourselves a disservice in assuming that using `wp-scripts` is good as we're proofing it while in reality, we tweak its config in a way that makes it different from what the users of `wp-scripts` do. I'd personally rather just embrace the fact that Gutenberg is a special case here and use webpack directly.

Reply:
Using webpack directly won't solve the issue fully. The shared webpack config which by default has this Babel integration built-in is something that needs to be tweaked.

Reply:
I think it's important to understand why we're using webpack at the root level of Gutenberg to understand the issue and what is the "common" webpack config Gutenberg setup has with wp-scripts setup.

The answer is simple, there's nothing really important to share between these two webpack configs aside the `DependencyExtractionWebpackPlugin` plugin for me. Is this sufficient to try to include checks in the common config instead of just duplicating the 4/5 lines that are common?